scalr/resource_scalr_workspace.go

```diff
@@ -167,9 +167,17 @@ func resourceScalrWorkspace() *schema.Resource {
 			},
 
 			"auto_queue_runs": {
-				Type:     schema.TypeBool,
+				Type:     schema.TypeString,
 				Optional: true,
-				Default:  nil,
+				ValidateFunc: validation.StringInSlice(
+					[]string{
+						string(scalr.AutoQueueRunsModeDefault),
+						string(scalr.AutoQueueRunsModeEnabled),
+						string(scalr.AutoQueueRunsModeDisabled),
+					},
+					false,
+				),
+				Computed: true,
```

Comment:
Why is this attribute  `Computed`? If write and apply configuration with `auto_queue_runs = "false"` and then remove `auto_queue_runs` argument from configuration and apply, will it be reset to `default`?

Reply:
No, only if it not present in configuration it will be set to "default"  by default

Reply:
@DayS1eeper updated, now by default is 'default' :) if user had originaly `auto_queue_runs = "disabled"` and then deletes it from configuration it will be set to 'default'